scvelo/tools/_em_model.py

```diff
@@ -126,6 +125,10 @@ def __init__(
         self._n_jobs = get_n_jobs(n_jobs=n_jobs)
         self._backend = backend
 
+    def _initialize_state_dict(self, adata, pars_names=None, key="fit"):
+        pars = _read_pars(adata, pars_names, key)
```

Comment:
Can you please refactor the code s.t. we do not call `_read_pars`? We'll remove `_em_model_core.py` and `_em_model_utils.py` in the future.

Reply:
Of course, this refactoring is done.